[2025-05-01T00:00:03.879+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_pipeline.extract_to_raw scheduled__2025-04-30T00:00:00+00:00 [queued]>
[2025-05-01T00:00:03.899+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_pipeline.extract_to_raw scheduled__2025-04-30T00:00:00+00:00 [queued]>
[2025-05-01T00:00:03.899+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-05-01T00:00:03.922+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): extract_to_raw> on 2025-04-30 00:00:00+00:00
[2025-05-01T00:00:03.945+0000] {standard_task_runner.py:60} INFO - Started process 15634 to run task
[2025-05-01T00:00:03.958+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'data_pipeline', 'extract_to_raw', 'scheduled__2025-04-30T00:00:00+00:00', '--job-id', '25', '--raw', '--subdir', 'DAGS_FOLDER/data_pipeline_dag.py', '--cfg-path', '/tmp/tmp5kuh0z8f']
[2025-05-01T00:00:03.965+0000] {standard_task_runner.py:88} INFO - Job 25: Subtask extract_to_raw
[2025-05-01T00:00:04.011+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-05-01T00:00:04.088+0000] {task_command.py:423} INFO - Running <TaskInstance: data_pipeline.extract_to_raw scheduled__2025-04-30T00:00:00+00:00 [running]> on host 59792d724231
[2025-05-01T00:00:04.250+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_pipeline' AIRFLOW_CTX_TASK_ID='extract_to_raw' AIRFLOW_CTX_EXECUTION_DATE='2025-04-30T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-04-30T00:00:00+00:00'
[2025-05-01T00:00:05.398+0000] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2025-05-01T00:00:05.399+0000] {base.py:83} INFO - Using connection ID 'postgres_default' for task execution.
[2025-05-01T00:00:31.569+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-05-01T00:00:31.592+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=data_pipeline, task_id=extract_to_raw, execution_date=20250430T000000, start_date=20250501T000003, end_date=20250501T000031
[2025-05-01T00:00:31.830+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-05-01T00:00:31.876+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
